.github/workflows/regression.yml

```diff
@@ -122,16 +122,13 @@ jobs:
       comparison-tag: ${{ steps.pr-metadata-merge-queue.outputs.COMPARISON_TAG || steps.pr-metadata-comment.outputs.COMPARISON_TAG }}
 
       # below are used in the experiment/analyze jobs
-      cpus: ${{ steps.system.outputs.CPUS }}
       memory: ${{ steps.system.outputs.MEMORY }}
       vector-cpus: ${{ steps.system.outputs.VECTOR_CPUS }}
 
       replicas: ${{ steps.experimental-meta.outputs.REPLICAS }}
       warmup-seconds: ${{ steps.experimental-meta.outputs.WARMUP_SECONDS }}
       total-samples: ${{ steps.experimental-meta.outputs.TOTAL_SAMPLES }}
```

Comment:
I'd lean towards dropping these as well - all have [analogs in global settings](https://github.com/DataDog/single-machine-performance/blob/main/terraform/per_team_infra/vector/main.tf#L46-L49)